dist/index.html

```diff
@@ -1,3 +1,6 @@
+//add new icons and fonts 
+
```

Comment:
It's comment only in PR. Though you can add a favicon in it.
`<link rel="shortcut icon" href="../path/favicon.ico">`